src/Fmt_ast.ml

```diff
@@ -1305,6 +1305,63 @@ and fmt_args ~first:first_grp ~last:last_grp c ctx args =
   in
   list_pn args fmt_arg
 
+and fmt_sequence c ?ext parens width xexp pexp_loc fmt_atrs =
+  let fmt_sep ?(force_break = false) xe1 ext xe2 =
+    let blank_line =
+      match c.conf.sequence_blank_line with
+      | `Preserve_one ->
+          let l1 = xe1.ast.pexp_loc.loc_end.pos_lnum in
+          let l2 = xe2.ast.pexp_loc.loc_start.pos_lnum in
+          l2 - l1 > 1
+      | `Compact -> false
+    in
+    let break =
+      if blank_line then fmt "\n@;<1000 0>"
+      else if c.conf.break_sequences || force_break then fmt "@;<1000 0>"
+      else if parens && Poly.(c.conf.sequence_style = `Before) then
+        fmt "@;<1 -2>"
+      else fmt "@;<1 0>"
+    in
+    match c.conf.sequence_style with
+    | `Before ->
+        break $ str ";"
+        $ fmt_extension_suffix c ext
+        $ fmt_or_k (Option.is_some ext)
+            (fmt_or parens "@ " "@;<1 2>")
+            (str " ")
+    | `Separator -> str " ;" $ fmt_extension_suffix c ext $ break
+    | `Terminator -> str ";" $ fmt_extension_suffix c ext $ break
+  in
+  let is_simple x = is_simple c.conf width x in
+  let break (_, xexp1) (_, xexp2) =
+    not (is_simple xexp1 && is_simple xexp2)
+  in
+  let elts = Sugar.sequence c.conf c.cmts xexp in
+  let first_ext =
```

Comment:
That variable is not needed after the assert so you could move the assert inside the match and transform that into an `()`.

Reply:
done